packages/functional-tests/lib/metrics.ts

```diff
@@ -123,5 +137,24 @@ export class MetricsObserver {
         }
       }
     });
+    this.subscribePage.page.on('response', (response) => {
+      if (
+        response.request().method() === 'POST' &&
+        response.request().url().includes('/metrics')
+      ) {
+        expect(response.status()).toBe(200);
+      }
+    });
```

Comment:
This asserts that the request validation was successful (this is where acquisition query parameters are first validated for example).